### ISSUE TITLE
how does it work with stack ?

### ISSUE THREAD

**Issue by rvion** [ranjitjhala/hover-tooltips-hdevtools#4]
Hello, is this compatible with stack + multiple package repo ?

**Comment by ranjitjhala** [comment]
I have been using it regularly "out of the box" with the following projects: 
- https://github.com/ucsd-progsys/liquid-fixpoint/blob/master/stack.yaml
- https://github.com/ucsd-progsys/liquidhaskell/blob/master/stack.yaml

Is that what you meant by multiple packages?

**Comment by rvion** [comment]
yes, thanks for the quick answer, I'll try again this evening, so.
